src/typescript.ts

```diff
@@ -80,8 +80,8 @@ export function generateTableInterface(
       }
     }
 
-    selectableMembers += `${jsdoc}${columnName}: ${tsType}${possiblyOrNull};\n`;
-    insertableMembers += `${jsdoc}${columnName}${insertablyOptional}: ${tsType}${possiblyOrNull};\n`;
+    selectableMembers += `${jsdoc}'${columnName}': ${tsType}${possiblyOrNull};\n`;
+    insertableMembers += `${jsdoc}'${columnName}'${insertablyOptional}: ${tsType}${possiblyOrNull};\n`;
```

Comment:
column name of 50-15-d doesn't project to an interface correctly, wrapping in quotes takes care of this